Sources/AsyncLocationKit/AsyncLocationManager.swift

```diff
@@ -45,6 +45,14 @@ public final class AsyncLocationManager {
         locationManager.desiredAccuracy = desiredAccuracy.convertingAccuracy
     }
     
+    public init(locationManager: CLLocationManager, desiredAccuracy: LocationAccuracy) {
+        self.locationManager = locationManager
+        self.locationManager.delegate = locationDelegate
```

Comment:
I get the error "Variable 'self.locationDelegate' used before being initialized" on this line

Reply:
sorry for this, this init was created for tests, I did not notice an error when pushing, corrected :)

Reply:
Thanks! Now it works just fine. And the solution to add the APPCLIP condition seems to work fine as well!

Reply:
Nice 😊

Reply:
No, I celebrated to early. Uploading to TestFlight now gives the same error as before. I guess the APPCLIP flag does not propagate down to the SPM frameworks :( Not sure how to solve it now. I guess I either have to clone and copy down the code for this framework and the FLAG will be respected, or just skip the framework for my AppClip target and loose all GPS functionality for my AppClip-target :(

Reply:
Can you use cocoapods?

Reply:
I've given up on Cocoapods a long time ago. SPM only now.

Reply:
I'll think about how to solve your problem)